maat-court-data-api/src/main/java/gov/uk/courtdata/repository/FdcItemsRepository.java

```diff
@@ -0,0 +1,14 @@
+package gov.uk.courtdata.repository;
+
+import gov.uk.courtdata.entity.FdcItemsEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface FdcItemsRepository extends JpaRepository<FdcItemsEntity, Integer> {
```

Comment:
Please can you include this new repository in the `Repositories` integration test utility class?
(Ideally we would have a architecture based unit test to validate this)

https://github.com/ministryofjustice/laa-maat-court-data-api/blob/de047b01d9def01bac364d2e8cf63dab3f867a9f/maat-court-data-api/src/test/java/gov/uk/courtdata/integration/util/Repositories.java